lms/services/__init__.py

```diff
@@ -23,6 +24,10 @@ def includeme(config):
     config.register_service_factory(
         "lms.services.basic_blackboard_api.factory", name="basic_blackboard_api_client"
     )
+    config.register_service_factory(
+        "lms.services.blackboard_api.blackboard_api_client_factory",
+        name="blackboard_api_client",
+    )
```

Comment:
The new `"blackboard_api_client"` service. Notice that `"basic_blackboard_api_client"` still exists as a separate service for now